cdn_lambda/functions/map_to_s3/map_to_s3.py

```diff
@@ -48,7 +54,7 @@ def handler(self, event, context):
             ScanIndexForward=False,
             KeyConditionExpression="web_uri = :u and from_date <= :d",
             ExpressionAttributeValues={
-                ":u": {"S": request["uri"]},
+                ":u": {"S": self.uri_alias(request["uri"])},
```

Comment:
The LOG.info a few lines above here is now lying (It will say "Querying <table_name> table for <uri>" where uri isn't what it's really querying for).  Can that please be fixed?

Ideally the logs contain enough info to judge whether the alias resolving is working correctly.

Reply:
This was intentional to represent the original request URI but I can change it.